app/services/import_mitigation.rb

```diff
@@ -0,0 +1,52 @@
+class ImportMitigation
+  # rubocop:disable LineLength
+  MITIGATION_CATEGORIES_FILEPATH = 'public/mocks/mitigation_categories.json'.freeze
+  MITIGATION_ACTIONS_FILEPATH = "#{CW_FILES_PREFIX}3. Mitigation/Table 27 - Additional mitigation actions in energy sector not in BUR1 - Sheet1.csv".freeze
+  FLAGSHIP_PROGRAMMES_FILEPATH = nil # FIXME: Still no data available
+  # rubocop:enable LineLength
+
+  def call
+    cleanup
+    create_categories
+    import_actions(S3CSVReader.read(MITIGATION_ACTIONS_FILEPATH))
+  end
+
+  private
+
+  def cleanup
+    Mitigation::FlagshipProgramme.delete_all
+    Mitigation::MitigationAction.delete_all
+    Mitigation::MitigationCategory.delete_all
+  end
+
+  def create_categories
+    json = JSON.parse(File.read(File.join(Rails.root, MITIGATION_CATEGORIES_FILEPATH)))
```

Comment:
Don't think this is a good approach, we're effectively hardcoding the existing themes. I think the themes should always come from the data files and be updatable in this way, e.g. if you take a look at "Table 15 - Implementation Progress of Actions whose mitigation effects have been estimated.xlsx" the theme is the first column, and that should be the case for all files which hold this data - see my comment about suggesting merging these files to a common format.